npm/webpack-dev-server/src/helpers/sourceRelativeWebpackModules.ts

```diff
@@ -180,11 +187,10 @@ export function sourceWebpack (config: WebpackDevServerConfig, framework: Source
 
 // Source the webpack-dev-server module from the provided framework or projectRoot.
 // If none is found, we fallback to the version bundled with this package.
-export function sourceWebpackDevServer (config: WebpackDevServerConfig, framework?: SourcedDependency | null): SourcedWebpackDevServer {
+export function sourceWebpackDevServer (config: WebpackDevServerConfig, webpackMajorVersion: 4 | 5, framework?: SourcedDependency | null): SourcedWebpackDevServer {
```

Comment:
Is 3 the version that is the fallback and why it isn't part of this `webpackMajorVersion` union here?

Reply:
luckily for us we don't support webpack v3, only v4 and v5. But with this PR we would support webpack-dev-server v3, v4, and v5.

the package currently ships with webpack-dev-server v4 but will source whatever the user has installed